SCClassLibrary/Common/UnitTesting/UnitTest.sc

```diff
@@ -148,22 +112,21 @@ UnitTest {
 
 	assertFloatEquals { |a, b, message = "", within = 0.0001, report = true, onFailure|
 		var details =
-			"Is:\n\t" + a +
-			"\nShould equal (within range" + within ++ "):\n\t" + b;
+		"Is:\n\t" + a +
+		"\nShould equal (within range" + within ++ "):\n\t" + b;
```

Comment:
This looked better how it was before IMO

Reply:
yes, that was automatic indentation.